Sources/SyncDataProviders/Bookmarks/BookmarksProvider.swift

```diff
@@ -148,7 +179,7 @@ public final class BookmarksProvider: DataProvider {
         }
 
         if let serverTimestamp {
-            lastSyncTimestamp = serverTimestamp
+            updateTimestamps(server: serverTimestamp, local: clientTimestamp)
```

Comment:
Here's when local timestamps are updated - they're updated at the end of the response handling, with the timestamp of making a request to the server (instantiated as `Date()` in SyncOperation).